desc/objectives/_coils.py

```diff
@@ -597,3 +597,208 @@ def compute(self, params, constants=None):
         data = tree_flatten(data, is_leaf=lambda x: isinstance(x, dict))[0]
         out = jnp.concatenate([dat["torsion"] for dat in data])
         return out
+
+
+class ToroidalFlux(_Objective):
+    """Target the toroidal flux in an equilibrium from a magnetic field.
+
+    This objective is needed when performing stage-two coil optimization on
+    a vacuum equilibrium, to avoid the trivial solution of minimizing Bn
+    by making the coil currents zero. Instead, this objective ensures
+    the coils create the necessary toroidal flux for the equilibrium field.
+
+    Parameters
+    ----------
+    eq : Equilibrium
+        Equilibrium that will be optimized to satisfy the Objective.
```

Comment:
The Equilibrium is assumed fixed and is not optimized, only the field, correct? Please state that here.